benchmarks/benchmarks/stats.py

```diff
@@ -249,6 +249,84 @@ def time_distribution(self, dist_name, method):
         self.method(*self.args, **self.kwds)
 
 
+class TrackContinuousRoundtrip(Benchmark):
+    # Benchmarks that track a value(s) for every distribution can go here
+    param_names = ['dist_name']
+    params = sorted(list(set([d[0] for d in distcont])))
+    dist_data = dict(distcont)
+
+    def setup(self, dist_name):
+        # MOST SETUP CODE COPIED FROM ABOVE `DistributionsAll` BENCHMARK #
+        self.dist = getattr(stats, dist_name)
+
+        if isinstance(self.dist, stats.rv_discrete):
+            raise NotImplementedError("Skipped b/c discrete")
+
+        self.shape_args = self.dist_data[dist_name]
+
+    def track_distribution_ppf_roundtrip(self, dist_name):
+        # Tracks the worst relative error of a couple of
+        # round-trip ppf -> cdf calculations.
+        vals = [0.001, 0.5, 0.999]
+
+        ppf = self.dist.ppf(vals, *self.shape_args)
+        round_trip = self.dist.cdf(ppf, *self.shape_args)
+
+        err_rel = np.abs(vals - round_trip) / vals
+        return np.max(err_rel)
+
+    def track_distribution_ppf_roundtrip_extrema(self, dist_name):
+        # Tracks the worst error (either absolute or relative) of a couple
+        # of round-trip ppf -> cdf calculations.
```

Comment:
I don't fully understand the meaning of "of a couple" here. `v` is scalar, and `self.dist_data` is just the shape parameters. Is it possible to clarify, or is this sort of left over from the other methods where v is array?

It also seems to me that this is just absolute error, not either.

Reply:
Yes, I think this was a holdout.